temp_file.py

```diff
@@ -58,8 +61,12 @@ def add_data(self, data: dict):
             self.close()
             raise ValueError('Expected data to have "navnelbnr" key')
 
+        dataCopy = data.copy()
+        dump = json.dumps(dataCopy['smiley_reports'])
+        dataCopy['smiley_reports'] = dump
```

Comment:
What is the logic behind this? Seems a bit weird to reassign, but maybe I'm missing something.

Reply:
To  be able to save the nested dict and list for the smiley reports in the csv file

Reply:
ah you mean it should be:
`dataCopy['smiley_reports'] = json.dumps(dataCopy['smiley_reports'])`

I was debugging and didn't change it back

Reply:
No no, I was just wondering about the reassignment :D
But ok, that makes sense